app/views/spree/shared/_reviews.html.erb

```diff
@@ -8,7 +8,5 @@
       <%= render 'spree/shared/review', review: review %>
     <% end %>
   <% end %>
-  <% if !Spree::Reviews::Config[:require_login] || spree_current_user %>
```

Comment:
@priyank-gupta right now you have to sign in to add a review, visitors don't even know they can add a review, really bad UX. 

New flow: users clicks -> redirected to sign up (if Spree::Reviews::Config[:require_login] == tue) -> more registrations, more reviews, better UX, less confused visitors. Plus it's aligns better with full page caching.